frontend/src/hooks/useReceivingCaptions.tsx

```diff
@@ -0,0 +1,48 @@
+import { Subscriber } from '@vonage/client-sdk-video';
+import { useState, useEffect, useMemo } from 'react';
+
+export type CaptionsType = {
+  caption: string;
+  isReceivingCaptions: boolean;
+};
+
+export type ReceivingCaptionsProps = {
+  subscriber?: Subscriber | null;
+};
+
+export type CaptionReceivedType = {
+  streamId: string;
+  caption: string;
+  isFinal: boolean;
```

Comment:
Do we need to document what `isFinal` means?  How about `streamId`?

Reply:
Any reason we can't follow the doxygen-style convention like https://github.com/Vonage/vonage-video-react-app/blob/main/frontend/src/utils/getAccessDeniedError/getAccessDeniedError.ts ?

<img width="1181" alt="image" src="https://github.com/user-attachments/assets/da2d3466-34c6-40ab-96a1-f39a5ccd53b2" />

Reply:
Likely other examples of this in your PR as well